Spigot-Server-Patches/0662-Workaround-for-visually-missing-health-on-join.patch

```diff
@@ -0,0 +1,18 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: HexedHero <6012891+HexedHero@users.noreply.github.com>
```

Comment:
```suggestion
From: Doc <nachito94@msn.com>
```

Stealing patches isn't nice.

Reply:
Wasn't my intention, I don't know who he was and didn't know his information nor where to even find it and I wouldn't have wanted to publish his email without consent.

Reply:
He published it himself. ^^ Thanks for fixing!